code/modules/clothing/gloves/miscellaneous.dm

```diff
@@ -166,7 +166,7 @@
 	var/mob/living/M = loc
 
 	if(M.a_intent in accepted_intents)
-		if(istype(M.mind.martial_art, /datum/martial_art/cqc))
+		if(M.mind.martial_art) || HAS_TRAIT(M, TRAIT_HULK))
```

Comment:
I forgot remove this in my last suggestion :upside_down_face:. That's why it's not compiling.
```suggestion
		if(M.mind.martial_art || HAS_TRAIT(M, TRAIT_HULK))
```